### PR TITLE
chore: provide translations for ExchangeDeveloperMode settings

### PR DESCRIPTION
**Pull request by gre**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - dev settings "Exchange Dev Mode"

### 📝 Description

this translates a dev settings introduced by https://github.com/LedgerHQ/ledger-live/pull/6475 and requested for translations

Capture d’écran 2024-05-13 à 11 12 58

### ❓ Context

- **JIRA or GitHub link**: 
  - https://ledgerhq.atlassian.net/browse/LIVE-12328
  - https://github.com/LedgerHQ/ledger-live/pull/6475

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/ExchangeDeveloperMode.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const ExchangeDeveloperMode = () => {
   };
 
   return (
-    <SettingsSectionRow title="Exchange Dev Mode" desc="Add exchange provider's info">
+    <SettingsSectionRow title={t("settings.developer.title")} desc={t("settings.developer.desc")}>
```
Comment (LucasWerey) [inline apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/ExchangeDeveloperMode.tsx:42]:
```suggestion
    <SettingsSectionRow title={t("settings.developer.exchangeDeveloperMode.title")} desc={t("settings.developer.exchangeDeveloperMode.desc")}>
```

File: apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/ExchangeDeveloperMode.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const ExchangeDeveloperMode = () => {
               isSearchable={false}
               onChange={handleProviderInfo}
               value={providerInfo}
-              placeholder="Exchange provider's info in CAL format"
+              placeholder={t("settings.developer.placeholder")}
```
Comment (LucasWerey) [inline apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/ExchangeDeveloperMode.tsx:52]:
```suggestion
              placeholder={t("settings.developer.exchangeDeveloperMode.placeholder")}
```

**Review by mcayuelas-ledger — APPROVED** [review]

**Review by cgrellard-ledger — APPROVED** [review]

**Review by mcayuelas-ledger — CHANGES_REQUESTED** [review]
@LucasWerey Is right :)

**Comment by gre** [comment]
oopsy, will fix and retest 👍

**Comment by gre** [comment]
Capture d’écran 2024-05-13 à 11 12 58

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**